ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/PatientDetailView.java

```diff
@@ -536,9 +541,12 @@ public void run() {
 				Messages.Patientenblatt2_showAddress) {
 			@Override
 			public void doRun() {
-				Kontakt a = Kontakt.load(((BezugsKontakt) inpZusatzAdresse.getSelection()).get(BezugsKontakt.OTHER_ID));
-				KontaktDetailDialog kdd = new KontaktDetailDialog(scrldfrm.getShell(), a);
-				kdd.open();
+				Kontakt a = Kontakt.load(
+					((BezugsKontakt) inpZusatzAdresse.getSelection()).get(BezugsKontakt.OTHER_ID));
+				KontaktDetailDialog kdd = new KontaktDetailDialog(scrldfrm.getShell(), a, bLocked);
+				if (kdd.open() == Dialog.OK) {
+					setPatient(ElexisEventDispatcher.getSelectedPatient());
```

Comment:
Bitte einbauen, wie in https://redmine.medelexis.ch/issues/6676 angegeben! Danke